Content.Client/GameObjects/Components/CloningPod/CloningPodWindow.cs

```diff
@@ -100,17 +103,36 @@ public void Populate(CloningPodBoundUserInterfaceState state)
             {
                 _scanManager = state.MindIdName;
                 BuildCloneList();
-                _lastUpdate = state;
             }
+            _lastUpdate = state;
 
-            var percentage = state.Progress / _cloningProgressBar.MaxValue * 100;
-            _progressLabel.Text = $"{percentage:0}%";
-
-            _cloningProgressBar.Value = state.Progress;
+            _cloningProgressBar.MaxValue = state.Maximum;
+            UpdateProgress();
             _mindState.Text = Loc.GetString(state.MindPresent ? "Consciousness Detected" : "No Activity");
             _mindState.FontColorOverride = state.MindPresent ? Color.Green : Color.Red;
         }
 
+        protected override void FrameUpdate(FrameEventArgs args)
+        {
+            base.FrameUpdate(args);
+            UpdateProgress();
+        }
+
+        private void UpdateProgress()
+        {
+            if (_lastUpdate == null)
+                return;
+            float simulatedProgress = _lastUpdate.Progress;
+            if (_lastUpdate.Progressing)
+            {
+                TimeSpan sinceReference = IoCManager.Resolve<IGameTiming>().CurTime - _lastUpdate.ReferenceTime;
+                simulatedProgress += (float) sinceReference.TotalSeconds;
+            }
+            var percentage = simulatedProgress / _cloningProgressBar.MaxValue * 100;
```

Comment:
The percentage goes over 100% when the cloned person doesn't transfer to the new body, so either this needs a Max calculation or the server needs to send another message when progress is complete.

Reply:
Fixed in 4adfc5d